packages/media/src/preloadVideo.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Preload video using a promise.
+ *
+ * @param {string} src Video source.
+ * @return {Promise} Video object.
+ */
+const preloadVideo = (src) => {
```

Comment:
Wonder if we can use this function here
https://github.com/google/web-stories-wp/blob/c3a3b997babddaae9a87ae6c47c8fbd1e65f9a47/packages/media/src/getFirstFrameOfVideo.js#L25-L31

https://github.com/google/web-stories-wp/blob/c3a3b997babddaae9a87ae6c47c8fbd1e65f9a47/packages/media/src/hasVideoGotAudio.js#L25-L28

Reply:
Also do we also need `preload` here?

Reply:
We need just the metadata here, so no need to preload the full video at that point, although the name `preloadVideo` can be a bit deceiving in this case. Maybe `preloadVideoMetadata` would be more accurate.

Reply:
Create a pr for preloadVideoMeta https://github.com/google/web-stories-wp/pull/8716

Reply:
@miina Makes sense. Could you rename yours to `preloadVideoMetadata`?